### PR TITLE
MediaAttachment: correct MediaAttachment width calculating

### LINKED ISSUE #796: MediaAttachment is out of block-quote's bounds
### Expected behavior

img_0261

### Actual behavior

img_0262

### Steps to reproduce the behavior

Insert image into block-quote (image must be wider than block-quote)

### PR DESCRIPTION
**Pull request by azone**
Fixes #796

To test:

Insert image or video into block-quote

### CODE REVIEW

File: Aztec/Classes/TextKit/MediaAttachment.swift [hunk 1]
```diff
@@ -298,22 +298,31 @@ open class MediaAttachment: NSTextAttachment {
...
         if let storage = textContainer?.layoutManager?.textStorage,
-           let paragraphStyle = storage.attribute(NSParagraphStyleAttributeName, at: charIndex, effectiveRange: nil) as? NSParagraphStyle {
-            padding += paragraphStyle.firstLineHeadIndent + paragraphStyle.tailIndent
+            let paragraphStyle = storage.attribute(NSParagraphStyleAttributeName, at: charIndex, effectiveRange: nil) as? NSParagraphStyle {
+            let attachmentString = storage.attributedSubstring(from: NSMakeRange(charIndex, 1)).string
+            let headIndent = storage.string.isStartOfParagraph(at: attachmentString.startIndex) ? paragraphStyle.firstLineHeadIndent : paragraphStyle.headIndent
+
+            padding += headIndent
+
+            if paragraphStyle.tailIndent > 0 {
```
Comment (SergioEstevao) [inline Aztec/Classes/TextKit/MediaAttachment.swift]: Can you explain why do we need this IF, in what scenarios the tailIdent can be positive or negative?

Comment (azone) [reply Aztec/Classes/TextKit/MediaAttachment.swift]: Here is **Discussion** of tailIndent from apple:

> If positive, this value is the distance from the leading margin (for example, the left margin in left-to-right text). If 0 or negative, it’s the distance from the trailing margin.
>
> For example, a paragraph style designed to fit exactly in a 2-inch wide container has a head indent of 0.0 and a tail indent of 0.0. One designed to fit with a quarter-inch margin has a head indent of 0.25 and a tail indent of –0.25.

Comment (SergioEstevao) [reply Aztec/Classes/TextKit/MediaAttachment.swift]: So could we rewrite this like this: 
```padding += abs(paragraphStyle.tailIndent) + abs(headIndent)```

**Comment by jleandroperez** [comment]
Closing in favor of #828.

Thanks **a lot** sir!!

**Comment by diegoreymendez** [comment]
@jleandroperez - Just double checking... this is the issue and #828 is the PR for this report.  Is that correct?

**Comment by jleandroperez** [comment]
@diegoreymendez Nope! #796 is the issue. #828 is Sergio’s fix Mark II, while this one is the original PR (AKA Fix Mark I).

**Pull request closed without merging**